lib/caiquenator_web/templates/page/index.html.eex

```diff
@@ -1,9 +1,9 @@
-<p class="lead text-center">
-  <%= @phrase %>
-</p>
+<div data-component class="text-center">
```

Comment:
We should use a specific name here. Otherwise, every `data-component` on the page is going to start the script. I just use `component` in the suggestion to exemplify 🙈.

Maybe it could be: `data-random-phrase` 🤔.

Reply:
Mindisgurpe.

Done